server/src/main/java/org/apache/iotdb/db/wal/utils/WALFileUtils.java

```diff
@@ -105,15 +105,15 @@ public static void ascSortByVersionId(File[] walFiles) {
 
   /**
    * Find index of the file which probably contains target insert plan. <br>
-   * Given wal files [ _0-0-1.wal, _1-5-0.wal, _2-5-1.wal, _3-12-1.wal, _4-12-1.wal ], details as
+   * Given wal files [ _0-0-1.wal, _1-5-0.wal, _2-5-1.wal, _3-12-0.wal, _4-12-1.wal ], details as
    * below: <br>
    * _0-0-1.wal: 1, 2, 3, -1, -1, 4, 5, -1 <br>
    * _1-5-0.wal: -1, -1, -1, -1 <br>
-   * _2-5-1.wal: 6, 7, 8, 9, -1, -1, -1, 10, 11, -1, 12, 12 <br>
-   * _3-12-1.wal: 12, 12, 12, 12, 12 <br>
+   * _2-5-1.wal: 6, 7, 8, 9, -1, -1, -1, 10, 11, -1, 12 <br>
+   * _3-12-0.wal: -1, -1, -1, -1, -1 <br>
```

Comment:
These comments don't need to be changed.

Reply:
Thanks for pointing out!